database/migrations/2021_10_31_143501_add_declined_at_column_to_articles_table.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDeclinedAtColumnToArticlesTable extends Migration
+{
+    public function up()
+    {
+        Schema::table('articles', function (Blueprint $table) {
+            $table->timestamp('declined_at')->nullable()->after('shared_at');
```

Comment:
@tomhatzer I want to make sure that when we deploy this, this migration won't fail. I *think* you need to explicitly set a default null here for existing records to be migrated over. Can you please try that out?

Reply:
As far as I know, for MariaDB and MySQL, a nullable column will default to NULL as value for existing database records. I just tested this with my local version by adding a new column to the migrations table as nullable. All existing items had the value NULL in them.

Reply:
@tomhatzer good enough for me, thanks! 👍